Dockerfile

```diff
@@ -13,29 +13,28 @@ RUN apt-get install --no-install-recommends -y build-essential && \
 
 # The python install is done in two steps to avoid re-installing all dependencies every
 # time the code changes
+# set the BUILD_CORE_SERVICE to non null to install additional service dependencies
+ARG BUILD_CORE_SERVICE
 COPY pyproject.toml poetry.lock README.rst CHANGES.rst Makefile /code/renku/
-WORKDIR /code/renku
-RUN poetry export --without-hashes -f requirements.txt --output /tmp/requirements.txt && \
-    pip install --no-cache-dir -r /tmp/requirements.txt && \
-    pip install --no-cache-dir poetry-dynamic-versioning==0.17.1
-
 COPY .git /code/renku/.git
 COPY renku /code/renku/renku
+WORKDIR /code/renku
+# hadolint ignore=SC2086
+RUN if [ -n "${BUILD_CORE_SERVICE}" ]; then export EXT_BUILD="-E service" ; fi && \
+    poetry export --without-hashes ${EXT_BUILD} -f requirements.txt --output /tmp/requirements.txt && \
+    pip install --no-cache-dir -r /tmp/requirements.txt
 
 # Set CLEAN_INSTALL to a non-null value to ensure that only a committed version of
 # renku-python is installed in the image. This is the default for chartpress builds.
 ARG CLEAN_INSTALL
 RUN if [ -n "${CLEAN_INSTALL}" ]; then git reset --hard ; fi && \
     make download-templates
 
-# set the BUILD_CORE_SERVICE to non null to install additional service dependencies
 ARG BUILD_CORE_SERVICE
-RUN if [ -n "${BUILD_CORE_SERVICE}" ]; then export EXT_BUILD="[service]" ; fi && \
-    pip wheel --wheel-dir /wheels ".${EXT_BUILD}" && \
+RUN pip wheel --wheel-dir /wheels ".[toil]" && \
```

Comment:
Just to be sure, we don't need `.[service]` here, right?

Reply:
I think this was a leftover from debugging things